### DISCUSSION TITLE
OLS Rebranding and Website Restructuring (including sub-website creation).

### DISCUSSION THREAD

**Discussion by NPDebs** [open-life-science/open-life-science.github.io#636] [General]
Hello everyone!

Here is some great news from OLS! We are bubbling with excitement to announce that we are in the process of **creating a new website** for our mentoring program, Open Seeds! 🌱 OLS is growing with different projects, beyond Open Seeds (for example, [NASA](https://openlifesci.org/posts/2023/05/05/NASA-grant-announcement/)). So, the current website has gradually become too cluttered and busy. As such, we've decided to split the current website between OLS and Open Seeds.

**Rebranding**
Over the past few months, a lot of effort has gone into **[creating a new brand identity](https://openlifesci.org/posts/2023/04/18/Introducing-the-new-OLS/)** for OLS and Open Seeds. This new website is part and parcel of the rebranding strategy. The purpose for creating it is to have a dedicated space for our mentoring initiatives, and provide a more seamless and enriching experience for all participants.

**Open Seeds Website**
In light of this exciting development, **we will be making some changes to our existing GitHub repository** to ensure that all resources related to the mentoring program find their rightful place in the new website's structure. To help you visualise this transition, we have prepared an **information architecture** diagram detailing where each file should be relocated. In this diagram, we are showing what resources will be found in **both websites** (in blue 🟦), as well as resources that are unique to OLS (💚) and Open Seeds (🟨) website.

**OLS Jekyll Theme**
A new theme has been created and is stored in this [ols-theme](https://github.com/open-life-science/ols-jekyll-theme) repository. It will be used across OLS websites, as it contains the aforementioned shared resources, as part of theme files. This boosts reusability and consistency.  
![An image showing the information architecture](https://github.com/open-life-science/open-life-science.github.io/assets/105166953/0e436abb-2be1-445d-89a9-d0196f0323f8)

**What should you expect?**
1. **New Domain Name:** Our current website "https://openlifesci.org/" will become "[we-are-ols.org](http://we-are-ols.org/)". In addition, we will have [we-are-ols.org](http://we-are-ols.org/) emails and various subdomains, such as [openseeds.we-are-ols.org](http://openseeds.we-are-ols.org/) for the mentoring program. This is a gradual process, so please feel free to ask if we’re ever unclear. Expect to see more we-are-ols in the future!

2. **New Repository:** This will house all the mentoring program's resources, including documentation, guidelines, and useful templates.

3. **File Migration:** To facilitate the smooth transition, we will be moving relevant files and folders from the existing repository to the new one. For instance, we’ve slowly been migrating logos and terminology for the brand assets to the new logo. Be confident that we will carefully manage this process to avoid any data loss or confusion.

4. **Maintaining Accessibility:** The goal is to ensure that all resources remain accessible to everyone. As such, we will leave information in the existing repository with clear redirection links to their new locations in the Open Seeds repository.

For a more detailed view on the entire process, check out the **[Github Project](https://github.com/orgs/open-life-science/projects/5)** here.

**Want to help us? Here's how:**
1. **Review and Feedback:** As we move forward with this change, we encourage everyone to review the information architecture diagram and share your thoughts. If you have any suggestions or spot anything that could be improved, please don't hesitate to let us know.

2. **Testing and Bug Reporting:**: Once the migration is complete, we will be counting on your support to help us test the new website thoroughly. If you come across any issues, bugs, or broken links, please report them immediately so that we can address them quickly.

**If you have any questions or need further clarification** about the upcoming changes, please feel free to reach out, or simply leave a comment under this discussion. Also, if you see any “open life science” rather than “OLS”, or if you see any of the old logo, please let us know! It’s not forbidden or anything, but we’re trying to get more consistent. Your feedback and insight is valuable!

Thank you!

**Comment by jilaga** [comment]
@NPDebs Do we make the checklist of issues here in the comments?

**Reply by NPDebs** [reply]
Yes, please. 😁

**Comment by NPDebs** [comment]
**MIGRATION CHECKLIST**

### **1.  Update theme**
- [ ] Remove resources we don't need
- [ ] Move files that will be shared by both repos to theme (Policies, Code of conduct, footer, people info, funding)
   - [ ] Policies
   - [ ] Code of conduct
   - [ ] Footer
   - [ ] People information
   - [ ] Funding
- [ ] Keep _data folder (except _cohorts subfolder)


### **2. Move resources to Open Seeds repo**
- [ ] Landing page
   - [ ] _About_ section
   - [ ] Application
   - [ ] Timeline'
   - [ ] Cohorts

- [ ] Move all cohort information (ols1-8)
   - [ ] _ols-x subfolder
   - [ ] ols-x.md files


### **3. Maintain OLS-specific resources in current OLS repo**
- [ ] Landing (index) page
- [ ] _About_ page
   - [ ] People page
   - [ ] Team and community
   - [ ] Funding and supporters
   - [ ] Partners
- [ ] Services
   - [ ] Consulting
- [ ] Stories (posts)

**Comment by jilaga** [comment]
Maybe we call it index page? i could update the IA to that too

**Reply by NPDebs** [reply]
I think landing page works as well. Maybe we can provide extra info in the bracket beside the name...just so everyone is clear. Make sense?

**Reply by jilaga** [reply]
Sure does!

**Comment by yochannah** [comment]
@NPDebs, @jilaga am I right in thinking this conversation thread could be closed?

**Comment by NPDebs** [comment]
Yes please. xD
